test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -506,6 +506,45 @@ public class CustomMappings : CustomMappingFactory<InputRow, OutputRow>
             }
         }
 
+        public class IrisData
+        {
+            public float Label { get; set; }
+            public float SepalLength { get; set; }
+            public float SepalWidth { get; set; }
+            public float PetalLength { get; set; }
+            public float PetalWidth { get; set; }
+        }
+
+        [Fact]
+        public void ReadAndWriteBinaryData() =>
+            BinaryData();
+
+        private void BinaryData()
+        {
+            // An array of iris data points
+            var dataArray = new[]
+            {
+                new IrisData { Label=1, PetalLength=1, SepalLength=1, PetalWidth=1, SepalWidth=1 },
+                new IrisData { Label=0, PetalLength=2, SepalLength=2, PetalWidth=2, SepalWidth=2 }
+            };
+
+            // Create the ML.NET context.
+            var context = new MLContext();
+
+            // Create the data view from an IEnumerable.
+            // This method will use the definition of IrisData to understand what columns there are 
+            // in the data view. However, the objects in ML.NET are only "promises" of data since 
+            // ML.NET operations are lazy. One way to get a look at the data is with Schema Comprehension.
+            // Refer to this document for more information - https://github.com/dotnet/machinelearning/blob/master/docs/code/SchemaComprehension.md
+            var data = context.Data.LoadFromEnumerable(dataArray);
+
+            // Use a FileStream to create a file. Use the stream and the data view in the "SaveAsBinary" method.
+            using (var stream = new FileStream("./iris.idv", FileMode.Create))
```

Comment:
>using (var stream = new FileStream("./iris.idv", FileMode.Create)) [](start = 12, length = 66)

You can register the temp file for deletion using `DeleteOutputPath`. It's done above like this:
`RunEndToEnd(mlContext, data, DeleteOutputPath("custom-model.zip"));`